framework/src/org/checkerframework/framework/type/AnnotationClassLoader.java

```diff
@@ -126,18 +124,16 @@ public AnnotationClassLoader(final BaseTypeChecker checker) {
                         Pattern.compile(Character.toString(DOT), Pattern.LITERAL)
                                 .split(packageName)));
 
-        // Only load annotations if requested.  This avoids issuing an error
-        // if the qual package contains an annotation that is not a qualifier,
-        // but the checker does not try to use it as a qualifier.
-        loadedAnnotations = null;
+        // Only load annotations if requested.
```

Comment:
This comment isn't really clear. Requested how?
Why is it initialized to `null` instead of to `getBundledAnnotationClasses()`?

Reply:
I've modified this part to just load the bundled annotation classes, thus the field will never be null.

Reply:
Update: some checkers like the index checker explicitly declares its supported qualifiers in each of its subcheckers, whist sharing a single qual directory across its subcheckers. If we load all of the bundled qualifiers here during the constructor of AnnotationClassLoader then it issues warnings for some of the qualifiers such as [@LengthOf](https://github.com/typetools/checker-framework/blob/master/checker/src/org/checkerframework/checker/index/qual/LengthOf.java) which requires that its `@Target` has `METHOD` in addition to `TYPE_USE`.

`getBundledAnnotationClasses()` is called by checkers that want to load and use its bundled annotations without explicitly declaring them. I'll clarify the comment to indicate this, but the initialization to null and subsequent replacement in `getBundledAnnotationClasses()` is intended.